akka-stream/src/main/scala/akka/stream/io/Framing.scala

```diff
@@ -0,0 +1,173 @@
+package akka.stream.io
+
+import java.nio.ByteOrder
+
+import akka.stream.scaladsl.Flow
+import akka.stream.stage.{ TerminationDirective, PushPullStage, SyncDirective, Context }
+import akka.util.{ ByteIterator, ByteStringBuilder, ByteString }
+
+import scala.annotation.tailrec
+
+object Framing {
+
+  def delimiter(delimiterBytes: ByteString, maximumFrameLength: Int): Flow[ByteString, ByteString, Unit] =
```

Comment:
`delimiterBytes: ByteString` is not good parameter name. `def delimiter(delimiter: ByteString, ` might look funny. `separator`?